excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/parser/model/DefinitionDataItemTest.java

```diff
@@ -14,45 +11,47 @@
 import java.time.ZoneId;
 import java.util.Date;
 
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 import static java.lang.Boolean.FALSE;
 import static java.lang.Boolean.TRUE;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsNull.nullValue;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 public class DefinitionDataItemTest {
 
     private DefinitionDataItem item;
 
-    @Before
+    @BeforeEach
     public void setup() {
         item = new DefinitionDataItem(SheetName.CASE_EVENT.toString());
     }
 
-    @Test(expected = MapperException.class)
+    @Test
     public void shouldFail_whenRequiredAttributeDoesNotExist() {
-        try {
+        MapperException ex = assertThrows(MapperException.class, () -> {
             item.findAttribute(ColumnName.ID);
-        } catch (MapperException ex) {
-            Assertions.assertThat(ex).hasMessageContaining(
-                "Couldn't find the column ID in the sheet CaseEvent");
-            throw ex;
-        }
+        });
+        Assertions.assertThat(ex).hasMessageContaining(
```

Comment:
Two different versions of `assertThat()` are used in this class (`org.hamcrest.MatcherAssert.assertThat` and `org.assertj.core.api.Assertions.assertThat`). Can this be simplified to use just one? The assertj one is the least frequently used one.

Reply:
Class simplified to use one version of assertThat().